functions-methods.js

```diff
@@ -33,4 +59,23 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+console.log("\nOpdracht 3:");
+
+function checkEmailValidity(emailAdress) {
+    const lengthEmailAdress = emailAdress.length;
+    if ((emailAdress.includes("@")) &&
+        !(emailAdress.includes(",")) &&
+        !(emailAdress[lengthEmailAdress - 1] === ".")) {
+        return true
+    } else {
+        return false;
+    }
```

Comment:
Wellicht dat je de sub uitkomsten nog los zou kunnen trekken naar variabelen, maar dat is dan meer voor de leesbaarheid. Verder zou je de uitkomst van de expressie meteen terug kunnen geven
```suggestion
    return emailAdress.includes("@") &&
        !emailAdress.includes(",") &&
        emailAdress[lengthEmailAdress - 1] !== ".";
```